tutorials/get-started-python/index.mdx

```diff
@@ -0,0 +1,453 @@
+    print(my_new_variable)
+    ```
+    
+    The following output displays:
+
+    ```
+    Hello world
+    ```
+
+    When we ask Python to print the variable, it displays the value of that variable! Note that this time we didn't use quotation marks in our print statement: these are only necessary for strings, not for variables or numbers.
+
+    <Message type="tip">
+
+    The variable we created holds **string** data. Let's look at just some of the different types of data we can store inside a variable, and how the data type affects the syntax of creating and assigning variables.
+
+    | Type    | Description                                                                                      | Example of variable assignment           |
+    |---------|--------------------------------------------------------------------------------------------------|------------------------------------------|
+    | int     | A whole number                                                                                   | `my_variable = 3`                          |
+    | float   | A decimal number                                                                                 | `my_variable = 2.667`                      |
+    | string  | A character, or sequence of characters. Represents text, not numbers to be used in calculation.  | `my_variable = "I saw 7 sheep"`            |
+    | boolean | A true or false value.                                                                           | `my_variable = True`                       |
+    | list    | A collection of values. These values could be ints, floats, strings, booleans etc, or a mixture. | `my_variable = [1, 7, "hello", 2.5, True]` |
+
+    With Python, there is no need to specify what type of variable you are creating. It automatically understands from the syntax of the creation statement (for example, whether there are quotation marks, whether a number has a decimal point) what type the variable should be. <br /><br /> These are not all the data types available in Python, but they are the main ones you'll need to get started with the basics.
+
+    </Message>
+
+    You can change the value of a variable from one thing to another. Next, we change `my_new_variable` so that instead of holding the string "Hello world", it now contains the float (decimal number) 3.5:
+
+3. Type the following command into the shell and hit enter:
+
+    ```
+    my_new_variable = 3.5
+    ```
+
+    Remember that no quotation marks should be used for number variables.
+
+4. Type the following command into the shell and hit enter:
+
+    ```
+    print(my_new_variable)
+    ```
+    
+    The following output displays:
+
+    ```
+    3.5
+    ```
+
+    Let's see how you can use Python to do mathematical operations on number variables. It's pretty simple!
+
+5. Type the following commands to see how you can assign different numbers to different variables, then create a new variable that adds them together:
+
```

Comment:
Maybe add that you should type each one separately because I copied and pasted everything at once thinking it was gonna work